### PR TITLE
Release notes patch

### PR DESCRIPTION
**Pull request by JackFratto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+
+### New Features
+ * Added support for connecting to a local Postres database
+ * Added a user interface for connecting to a local database
+ * Added a button to display all stored procedures from a local database
+ 
+### Bug Fixes
+ * None
+### Known Issues
+ * Pressing the connect button infinitley establishes a connection with the Postgres database
```
Comment (adityavidyadharan) [inline README.md:19]: Currently, the connection configuration is a hardcoded set of values which is a known issue.

**Review by adityavidyadharan — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+
+Licenses under [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
+
+# Release Notes
+
+## Version 0.1.0
+
+### New Features
+
+- Added support for connecting to a local Postres database
```
Comment (alex-hobby) [inline README.md:13]: "Postgres"

File: README.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+- Added a user interface for connecting to a local database
+- Added a button to display all stored procedures from a local database
+
+### Bug Fixes
+
+- None
+
+### Known Issues
+
+- Pressing the connect button infinitley establishes a connection with the Postgres database
```
Comment (alex-hobby) [inline README.md:23]: "infinitely"

File: README.md [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+
+# Release Notes
+
+## Version 0.1.0
+
+### New Features
+
+- Added support for connecting to a local Postres database
+- Added a user interface for connecting to a local database
+- Added a button to display all stored procedures from a local database
```
Comment (alex-hobby) [inline README.md:15]: Maybe "added a dropdown"?

File: README.md [hunk 5]
```diff
@@ -1,3 +1,28 @@
+# Stored Procedure Unit Tester
+
+An appplication to unit test stores procedures among databases
```
Comment (alex-hobby) [inline README.md:3]: "An application to unit test stored procedures inside databases"

**Review by alex-hobby — APPROVED** [review]
👍

**Review by adityavidyadharan — APPROVED** [review]
Changes look good now

**Pull request merged**